express-mock-af/routes/consumption-reporting.js

```diff
@@ -0,0 +1,41 @@
+var express = require('express');
+var router = express.Router();
+
+var querystring = require('querystring');
+var querystring = require('body-parser');
+
+/* GET users listing. */
+router.post('/:aspId', function(req, res, next) {
+  const aspId = req.params.aspId;
+  res.send("respond a resource for consumption-reporting with post for aspId:" + aspId);
+  //res.send(`respond a resource for consumption-reporting with post for aspId:${aspId}`);
+  
+  console.log("1111 query");
```

Comment:
Remove all of these log outputs

Reply:
Note that the _aspId_ parameter is due to be replaced by _provisisioningSessionId_ in TS 26.512 V17.6.0.

See 5G-MAG/Standards#76 for links to Rel-17 **CR0037**r2. This will hopefully be approved at SA#101 Plenary next week.

Reply:
done